formats/detailed_guide/publisher/links.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "lead_organisations"
+  ],
+  "properties": {
+    "related_guides": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "alternative_format_provider_organisation": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "related_mainstream": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "additional_related_mainstream": {
```

Comment:
This would be better handled as a single links for both related and additional related, then a `related_mainstream_content` field in `details` which specifies the order.